tests/by-util/test_more.rs

```diff
@@ -32,3 +38,15 @@ fn test_more_dir_arg() {
             .usage_error("'.' is a directory.");
     }
 }
+
+#[test]
+#[cfg(target_family = "unix")]
+fn test_more_invalid_file_perms() {
+    if std::io::stdout().is_terminal() {
+        let (at, mut ucmd) = at_and_ucmd!();
+        let permissions = Permissions::from_mode(0o244);
+        at.make_file("invalid-perms.txt").metadata().unwrap();
```

Comment:
Retrieving the metadata is unnecessary as we don't use them.
```suggestion
        at.make_file("invalid-perms.txt");
```